k8s/perftest/common/ccd/data-store-api.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: ccd
   annotations:
     flux.weave.works/automated: "true"
-    flux.weave.works/tag.java: glob:prod-*
+    flux.weave.works/tag.java: glob:demo-*
```

Comment:
do we've image tags with `demo-xxx` ?

Reply:
i guess you need to use exact tag of 17.2 release cut and disable `flux.weave.works/automated: "true"`